app/views/candidate_interface/unsubmitted_application_form/show.html.erb

```diff
@@ -53,14 +53,23 @@
 
       <ul class="app-task-list govuk-!-margin-bottom-8">
         <li class="app-task-list__item">
-          <%= render(
-            TaskListItemComponent.new(
-              text: @application_form_presenter.references_link_text,
-              completed: @application_form_presenter.enough_references_provided?,
-              path: @application_form_presenter.references_path,
-              custom_status: @application_form_presenter.references_in_progress? ? 'In progress' : nil
-            )
-          ) %>
+          <div class="app-task-list__content">
+            <%= render(
+              TaskListItemComponent.new(
+                text: @application_form_presenter.references_link_text,
+                completed: @application_form_presenter.enough_references_provided?,
+                path: @application_form_presenter.references_path,
+                custom_status: @application_form_presenter.references_in_progress? ? 'In progress' : nil
+              )
+            ) %>
+          </div>
+          <div class="app-task-list__content">
+            <%= render(
+              TaskListItemReferencesComponent.new(
+                references: @application_form_presenter.references,
+              )
+            ) %>
+          </div>
```

Comment:
Can we have this additional `div.app-task-list__content` appear only for the references section?

Reply:
This section of code is only for the references section but I've moved the `<div>` inside the component and will only render it when there is 1 or more references to show.

Reply:
Perfect. Perhaps we can do the same for `TaskListItemComponent`, to be consistent?